piano-hero/mobile/src/main/java/com/novoda/pianohero/GameMvp.java

```diff
@@ -0,0 +1,38 @@
+package com.novoda.pianohero;
+
+interface GameMvp {
+
+    interface Model {
+
+        RoundViewModel startGame();
+
+        RoundViewModel onNotesPlayed(Callback callback, Note... note);
+
+        interface Callback {
+            void onSequenceComplete();
+        }
+    }
+
+    interface View {
+
+        void showRound(RoundViewModel viewModel);
+
+        void showGameComplete();
+
+        void showError();
+
+        void showKeyboard();
```

Comment:
Who do you expect to implement this `View`, I see it platform agnostic. So `showKeyboard()` here is not really platform agnostic. Maybe we're missing something. What's the domain action of this command if we're not thinking platform specific?

Also, being in `mobile` it doesn't force you to think about it as platform agnostic.